File: src/main/java/com/tom/guess/GameFor.java
package com.tom.guess;

import java.util.Random;
import java.util.Scanner;

public class GameFor {
    public static void main(String[] args) {

        /*for(int i=5;i>=1;i--){
            System.out.println(i);
        }*/

        Scanner scanner = new Scanner(System.in);
        int secret = new Random().nextInt(10)+1;
        System.out.println(secret);
        for(int i=1;i<=4;i++){
            System.out.print("Please enter a number :("+i+"/4) ");
        int number = scanner.nextInt();
        System.out.println("第 "+i+" 次 "+number);
        if(number > secret){
            System.out.println("lower");
        }else if(number < secret){
            System.out.println("higher");
        }else {
            System.out.println("Great!, the number is "+number);
            break;
        }

        }
        System.out.println("結束");
    }
}

File: src/main/kotlin/com/tom/GameFor.kt
package com.tom

import java.util.*

fun main() {
     val secret = Random().nextInt(10)+1
    val scanner = Scanner(System.`in`)
    println(secret)
    for(i in 1..4){
        print("Please enter a number (${i}/4): ")
        var number = scanner.nextInt()
        println("第${i}次 $number")
        if(number > secret){
            println("lower")
        }else if(number<secret){
            println("higher")
        }else{
            println("Great , the number is "+secret)
            break
        }
    }
}